listenbrainz_spark/recommendations/train_models.py

```diff
@@ -50,6 +52,27 @@ def compute_rmse(model, data, n):
       .values()
     return sqrt(predictionsAndRatings.map(lambda x: (x[0] - x[1]) ** 2).reduce(add) / float(n))
 
+def generate_model_id():
+    """ Generate best model id.
```

Comment:
best

Reply:
This function is generating model ids for many models, so I feel that the docstring should be corrected.